boards/OPENMV_AE3/omv_boardconfig.h

```diff
@@ -47,6 +47,9 @@ extern unsigned char OMV_BOARD_UID_ADDR[12];    // Unique address.
 // GPU configuration
 #define OMV_GPU_ENABLE                  (CORE_M55_HP)
 
+// Software ISP configuration
+#define OMV_GAMMA_TABLE_ENABLE          (1)
```

Comment:
This should now be an IMLIB define: `IMLIB_ENABLE_GAMMA_LUT`

Reply:
Done